renderer/components/onboard/__tests__/QuizSteps.test.js

```diff
@@ -0,0 +1,108 @@
+      <QuizSteps
+        onClose={toggleQuiz}
+        onDone={onQuizComplete}
+        questionList={[
+          <FormattedMessage
+            key="Onboarding.PopQuiz.1.Question"
+            id="Onboarding.PopQuiz.1.Question"
+          />,
+          <FormattedMessage
+            key="Onboarding.PopQuiz.2.Question"
+            id="Onboarding.PopQuiz.2.Question"
+          />,
+        ]}
+        actuallyList={[
+          <FormattedMessage
+            key="Onboarding.PopQuiz.1.Wrong.Paragraph"
+            id="Onboarding.PopQuiz.1.Wrong.Paragraph"
+          />,
+          <FormattedMessage
+            key="Onboarding.PopQuiz.2.Wrong.Paragraph"
+            id="Onboarding.PopQuiz.2.Wrong.Paragraph"
+          />,
+        ]}
+      />
+    )
+  })
+
+  afterEach(() => {
+    onQuizComplete.mockClear()
+  })
+
+  test('User Story with correct answers', async () => {
+    const yesButton1 = screen.getByText(English['Onboarding.PopQuiz.True'])
+    fireEvent.click(yesButton1)
+    const lottiePlayerFirst = screen.getByTestId('quiz-steps-animation')
+    await waitForElementToBeRemoved(lottiePlayerFirst, { timeout: 2000 })
+    const heading = screen.getByText(English['Onboarding.PopQuiz.2.Title'])
+    const yesButton2 = screen.getByText(English['Onboarding.PopQuiz.True'])
+    expect(heading).toBeInTheDocument()
+    fireEvent.click(yesButton2)
+    const lottiePlayerSecond = screen.getByTestId('quiz-steps-animation')
+    await waitForElementToBeRemoved(lottiePlayerSecond, { timeout: 2000 })
+    expect(onQuizComplete).toHaveBeenCalledTimes(1)
+  })
+
+  test('User Story with incorrect answers', async () => {
+    const falseButton1 = screen.getByText(English['Onboarding.PopQuiz.False'])
+    fireEvent.click(falseButton1)
+    const lottiePlayer1 = screen.getByTestId('quiz-steps-animation')
+    await waitForElementToBeRemoved(lottiePlayer1, { timeout: 2000 })
+    const warningParagraph1 = screen.getByText(
+      English['Onboarding.PopQuiz.1.Wrong.Paragraph']
+    )
+    const continueButton1 = screen.getByText(
+      English['Onboarding.PopQuiz.Wrong.Button.Continue']
+    )
+    expect(warningParagraph1).toBeInTheDocument()
+    fireEvent.click(continueButton1)
+    const quizHeading2 = await screen.findByText(
+      English['Onboarding.PopQuiz.2.Title']
+    )
+    expect(quizHeading2).toBeInTheDocument()
+    const falseButton2 = screen.getByText(English['Onboarding.PopQuiz.False'])
+    fireEvent.click(falseButton2)
+    const lottiePlayer2 = screen.getByTestId('quiz-steps-animation')
+    await waitForElementToBeRemoved(lottiePlayer2, { timeout: 2000 })
+    const warningParagraph2 = screen.getByText(
+      English['Onboarding.PopQuiz.2.Wrong.Paragraph']
+    )
+    const continueButton2 = screen.getByText(
+      English['Onboarding.PopQuiz.Wrong.Button.Continue']
+    )
+    expect(warningParagraph2).toBeInTheDocument()
+    fireEvent.click(continueButton2)
+    expect(onQuizComplete).toHaveBeenCalledTimes(1)
+  })
+})
```

Comment:
Wondering if it is useful to consider writing tests for all 4 combinations of button clicks. e.g [True, False] and [False, True]. What do you think? @Eddges

Reply:
I do not see any issue in that. It would rather enhance confidence in the tests. The tests are written for all four combinations now.